docs/extensibility/form-section.md

```diff
@@ -286,7 +286,7 @@ CodeEditor widgets render a versatile code editor that can be used to edit any v
 #### Widget-specific parameters
 
 - **language** - a [JSONata](jsonata.md) expression resolving the desired language. It has access to the `$root` variable, containing the entire resource.
-- **inputInfo** - a string below the input field that shows how to fill in the input.
+- **inputInfo** - a string below the input field that shows how to fill in the input. Links can be provided, {{ [`name`] (`link`) }} and optionally, a `name` instead of the `link` can be displayed.
```

Comment:
```suggestion
- **inputInfo** - a string below the input field that shows how to fill in the input. You can use the {{ [`name`] (`link`) }} format to display a `name` instead of a `link.
```